Mathlib/Topology/Algebra/Group/Basic.lean

```diff
@@ -1252,6 +1252,44 @@ theorem subset_interior_smul : interior s • interior t ⊆ interior (s • t)
 
 end ContinuousConstSMul
 
+section ContinuousSMul
+
+variable [TopologicalSpace α] [TopologicalSpace β] [Group α] [MulAction α β] [ContinuousInv α]
+  [ContinuousSMul α β] {s : Set α} {t : Set β}
+
+@[to_additive]
+theorem IsClosed.smul_left_of_isCompact (ht : IsClosed t) (hs : IsCompact s) :
```

Comment:
Is the right version `IsClosed.smul_right_of_isCompact` true? If yes, it would be worth proving. If not, it would be worth adding a comment to highlight the difficulty.

Reply:
Well it works if given `x` and `y` in the same orbit, we have a continuous (in `y`, `x` is fixed) way to choose an actual `g` such that `x = gy`. That would be the case for an `AddTorsor`, but there's no nice way to state it in general. I'll add a comment.